bootstrap-core/src/test/java/de/agilecoders/wicket/core/markup/html/bootstrap/form/FormGroupTest.java

```diff
@@ -166,7 +166,39 @@ void formGroupSubmitValidation() {
         formTester.submit();
 
         tester().assertLabel("form:id:error", "&#039;value&#039; is required.");
-        tester().assertContains("class=\".*is-invalid.*\""); //assert error CSS class is present
+        tester().assertContains("class=\"invalid-feedback\""); //assert error CSS class is present
+    }
+
+    @Test
+    void formGroupSubmitValidationSuccess() {
+
+        Model<FormData> model = Model.of(new FormData("test"));
+        model.getObject();
```

Comment:
Is this call really needed here ?

Reply:
I also wasn't sure about the model creation and `model.getObject();`call - but just copied the code from the other tests in `FormGroupTest.java` and added the required adjustments for success validation (`new FormData("test")`).

I can simplify the model creation, if you like. But adjusting the other tests, that don't belong to form group validation, might be better placed in a separate PR?